package.json

```diff
@@ -6,7 +6,7 @@
   "repository": "https://github.com/kubesail/deploy-node-app",
   "license": "MIT",
   "engines": {
-    "node": ">=10"
+    "node": ">8"
```

Comment:
CRA uses >=8, https://github.com/facebook/create-react-app/blob/master/packages/create-react-app/package.json#L15

Based on the number of reports, I'm thinking we should target the same. Does NPX check the `engines` block before executing? It doesn't seem to when its just `npm link`ed, but I haven't published a version to verify that.

Reply:
https://github.com/kubesail/deploy-node-app/compare/node-8-support?expand=1 is a start on supporting node 8. Shouldn't be too much more work to get there